app/views/sessions/leaderboard.erb

```diff
@@ -1,19 +1,40 @@
-<h2> Leader Board</h2>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href=
+"https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">     
+    <script src=
+"https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js">
+    </script>     
+    <script src=
+"https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js">
+    </script>     
+    <script src=
+"https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js">
+    </script>
+</head>
+<body style="background-color:#C0C0C0">
 
-<table>
+<div class="jumbotron text-center" style="color:#3399ff">
+  <h2>Leaderboard</h2>
+</div>
+
+<table class="table table-striped">
```

Comment:
I recommend wrapping the table in a div container, so the table has some padding on the sides.